### ISSUE TITLE
Can not find the transitions in ember-cli

### ISSUE THREAD

**Issue by mrinterweb** [ef4/liquid-fire#212]
I am using liquid-fire 0.17.1, and I can not get ember-cli to recognize the transitions. I get the following error:

`Cannot read property 'transitionFor' of undefined`

I even tried the exact sample app/transitions.js from the documentation, and my app still can not find the definitions. The error happens on line 61 of liquid-outlet.js
`var transition = this.get("transitions").transitionFor(this, oldView, newView, this.get("use"), firstTime);`

I see my transitions being compiled to app/dfw/transitions.js ("dfw" is the name of my app) as:
```
define('dfw/transitions', ['exports'], function (exports) {

  'use strict';

  exports['default'] = function () {
    this.transition(this.fromRoute("people.index"), this.toRoute("people.detail"), this.use("toLeft"), this.reverse("toRight"));
  };

});
```

Could this be an issue with my "dfw" namespace and liquid-fire not using the namespace?
EDIT: I'm using ember-cli version 0.2.0 and ember 1.10.

**Comment by ef4** [comment]
I've been trying to reproduce this but I haven't been able to.

From the description, the problem is not with finding individual transitions, it's that the `transitions` property is missing. It's supposed to be the dependency-injection global transition map.

**Comment by mrinterweb** [comment]
I am also trying to reproduce the issue with a new ember-cli app. As of now, I have not been able to reproduce the issue. I'll keep looking.

**Comment by mrinterweb** [comment]
I believe I found what my problem was. I was using some older style of initializers and I was not calling. 

`loadInitializers(App, config.modulePrefix)` in app.js

I think the error with transitions not being injected was a red herring caused by my app not properly initializing. I am porting a fairly large ember app from rails asset pipeline to ember-cli, and I am learning ember-cli conventions the hard way. Now that I have corrected my initializers, my app loads without JS errors. Sorry for the trouble you went through attempting to reproduce the issue. Should this error pop-up for someone else, they should take a look at their initializers and compare them to how ember-cli generates initializers.
